static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -73,14 +73,15 @@ define([
             },
             AU: {
                 membership: {
-                    messageText: 'We need you to help support our fearless independent journalism. Become a Guardian Australia Member for just $100 a year.',
+                    messageText: 'We need you to help support our fearless independent journalism. Become a Guardian Australia member for just $10 a month.',
```

Comment:
https://trello.com/c/nLAs3Qup/258-engagement-banner-test-8-aus-end-test-and-make-fearless-10-the-new-control